### PR TITLE
RFE: clarify seccomp_rule_add syscall translation behavior

### PR DESCRIPTION
**Pull request by Xyene**
Refs #259.

### CODE REVIEW

File: doc/man/man3/seccomp_rule_add.3 [hunk 1]
```diff
@@ -147,7 +147,11 @@ value directly using the standard
 values, in order to ensure proper operation across multiple architectures it
 is highly recommended to use the
 .BR SCMP_SYS ()
-macro instead.  See the EXAMPLES section below.
+macro instead.  See the EXAMPLES section below.  If using
+.B __NR_syscall
+values in a context containing non-native architectures, the syscall
+corresponding to the value in the native architecture will be used in all
+non-native architectures.
```
Comment (pcmoore) [inline doc/man/man3/seccomp_rule_add.3:154]: I not sure that reads quite right, how about the following?

> ... it is highly recommended to use the SCMP_SYS() macro instead, see the EXAMPLES section below.  It is also important to remember that regardless of the architectures present in the filter, the syscall numbers passed to the libseccomp rule add functions should always be in the context of the native architecture.

Comment (pcmoore) [reply doc/man/man3/seccomp_rule_add.3:154]: @Xyene what did you think of the substitute text above?

Comment (Xyene) [reply doc/man/man3/seccomp_rule_add.3]: Sorry about the delay, I got DOS'd by real-life stuff -- I've committed a slightly reworded variant of your suggested text, emphasizing that libseccomp is performing some "interpreting" of the given syscall value.

**Comment by pcmoore** [comment]
Thanks for getting back to this, I definitely understand competing priorities and I can assure you we appreciate the help regardless of where libseccomp falls in your personal priority list :)

This  updated text looks good to me, any objections @drakenclimber?

**Comment by drakenclimber** [comment]
Looks good to me :+1: 

`Acked-by: Tom Hromatka <tom.hromatka@oracle.com>`

**Comment by pcmoore** [comment]
Merged via fa6264bd9d674d5bd141d1153ed1d77e654b4af1, thanks!

**Pull request closed without merging**